From 9095f3aa546359281e98657b12c210e835cc7f48 Mon Sep 17 00:00:00 2001
From: Jason Hung <jhung@globalscaletechnologies.com>
Date: Thu, 26 Apr 2018 10:23:51 +0800
Subject: [PATCH 7/8] btusb: disable realtek bluetooth support

since the btrtl driver is conflict with realtek btusb driver,
so disabled this configuration and using external realtek btusb
driver.
---
 arch/arm64/configs/gti_espressobin_v6_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/gti_espressobin_v6_defconfig b/arch/arm64/configs/gti_espressobin_v6_defconfig
index d6c960f..4730109 100644
--- a/arch/arm64/configs/gti_espressobin_v6_defconfig
+++ b/arch/arm64/configs/gti_espressobin_v6_defconfig
@@ -323,7 +323,7 @@ CONFIG_BT_DEBUGFS=y
 
 CONFIG_BT_HCIBTUSB=m
 CONFIG_BT_HCIBTUSB_BCM=y
-CONFIG_BT_HCIBTUSB_RTL=y
+# CONFIG_BT_HCIBTUSB_RTL is not set
 
 CONFIG_BT_HCIUART=m
 CONFIG_BT_HCIUART_BCSP=y
-- 
1.9.1

